bones/dateBone.py

```diff
@@ -79,9 +79,7 @@ def singleValueFromClient(self, value, skel, name, origData):
 			:returns: str or None
 		"""
 		rawValue = value
-		if not rawValue:
-			return None, [ReadFromClientError(ReadFromClientErrorSeverity.Empty, name, "No value selected")]
-		elif str(rawValue).replace("-", "", 1).replace(".", "", 1).isdigit():
+		if str(rawValue).replace("-", "", 1).replace(".", "", 1).isdigit():
```

Comment:
I think you missed ":", or not?

Reply:
No, this is a timestamp. We'll just remove minus and the first decimal point to be able to check if every remaining character is a digit.